src/App.jsx

```diff
@@ -27,8 +75,15 @@ const App = () => {
       <h1>
         Todo List
       </h1>
-      <AddTodoForm onAddTodo={addTodo} />
-      <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
+      {isError
```

Comment:
Let's refactor this. Nested ternaries are not the way to go.

Why not 

```js
{ isError && (<p>{errorMessage}</p>) }
{ isLoading
  ? (<p>Loading...</p>)
   : (
       <>
         <AddTodoForm onAddTodo={addTodo} />
           <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
        </>
)}
```

Reply:
Good question... I have the following ideas:
1. Use attribute 'disabled' like in 'disabled ={isLoading || isError}'
2. Set 'style-"display: {displayStatus}" and set it to 'none' or 'block' depending on '(isLoading || isError)'. For that I will have to use yet another '[displayStatus, setDisplayStatus]'. I do not know if it is ok to have many 'useState', my list of them is growing already
3. I can also pass 'hidden' attribute (same like 'disabled') and make it 'hidden ={(isLoading || isError)}' in props
4. In Google I also found 'useSpring' where I can set 'opacity' to '+!(isLoading || isError)', ie 1 or 0. This will look like below. But I have not tried it yet:
```
const SpringIn = ({ children }) => {
  const props = useSpring({
    opacity: +!(isLoading || isError),
    from: { opacity: +(isLoading || isError)},
    config: { mass: 10, tension: 10, friction: 10 }
  });
  return <animated.div style={props}>{children}</animated.div>;
};
```
5. Also with Google help (but it is similar to my idea from #1), I can add 'isLoading' and 'isError' to the props of 'AddTodoForm' and 'TodoList'. Inside those components I can use the check:
`if (isLoading || isError) return null;`
6. Or are you thinking about something else that I have not thought about?

Reply:
Easier to do this:

```js
if (isError) {
  return (<p>{errorMessage}</p>)
}

 return (
    <>
      <h1>
        Todo List
      </h1> 

//... rest of code
```

Reply:
I did not think about that! Simple and elegant! Thank you